src/components/tx/TxData.js

```diff
@@ -901,9 +915,14 @@ class TxData extends React.Component {
           <div className="summary-balance-info">
             <h2 className="details-title">Overview</h2>
             <div className="summary-balance-info-container">
-              <div className="address-container-title">Type</div>{' '}
-              {hathorLib.transactionUtils.getTxType(this.props.transaction)}{' '}
-              {isNFTCreation() && '(NFT)'} <TxMarkers tx={this.props.transaction} />
+              <div className="address-container-title">Type</div> {getTxType()}{' '}
+              {isNFTCreation() && '(NFT)'}
+              {isBlueprint() && (
+                <Link to={`/blueprint/detail/${this.props.transaction.hash}`}>
+                  &nbsp;(see details)
```

Comment:
question(non-blocking): Did you try with `{' '}` like we have in the other places instead of `&nbsp;`?

Reply:
Tried, but whitespace was not showing